spec/helpers/shared_search_helper_spec.rb

```diff
@@ -5,12 +5,16 @@
 
   context "shared search records" do
     let(:cname) { 'hyku-me.test' }
-    let(:account) { build(:account, cname: cname) }
+    let(:account) { build(:search_only_account, cname: cname) }
```

Comment:
ideally would like to include a context for when account is not shared_search but I don't think I'll get back to this in time. I need to take care of some other issues before PTO! cc @orangewolf